### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by gitferry**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -8,12 +8,82 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
...
+8.  in violation of applicable laws, rules or regulations in your relevant jurisdiction;
+9.  that circumvents any sanctions or export controls targeting you or the country or territory where
+    you are located; or
+10. in any activity that transmits, exchanges, or is otherwise supported by the direct or indirect
+    proceeds of criminal or fraudulent activity.
+
+Any production use of the Licensed Work by you confirms your agreement to the foregoing limitations,
+and your understanding and agreement that they are limitations and not restrictions.
 
 Change Date:          2027-05-15 (March 15th, 2027)
```
Comment (vitsalis) [inline LICENSE:88]:
```suggestion
Change Date:          2027-03-15 (March 15th, 2027)
```

File: go.mod [hunk 2]
```diff
@@ -7,9 +7,9 @@ toolchain go1.22.4
 require (
 	cosmossdk.io/math v1.3.0
 	github.com/avast/retry-go/v4 v4.5.1
-	github.com/babylonlabs-io/babylon v0.9.0
+	github.com/babylonlabs-io/babylon v0.9.2
```
Comment (vitsalis) [inline go.mod:10]: Do we need to update the dependency here? Seems bit like a last min change

Comment (gitferry) [reply go.mod:10]: Nope if we don't need all the dependencies to have the latest LICENSE

File: LICENSE (hunk 1 above)
Comment (gitferry) [reply LICENSE:88]: what's the meaning of this date?

File: go.mod (hunk 2 above)
Comment (gitferry) [reply go.mod]: reverted all the dependency bump

**Review by vitsalis — APPROVED** [review]

**Pull request merged**